.github/workflows/approve-to-run-ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Meshery-Operator CI
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    branches: [ master ]
+
+jobs:
+    golangci-lint:
+        strategy:
+          matrix:
+            platform: [ubuntu-22.04]
+            go-version: [1.19.x]
+        runs-on: ${{ matrix.platform }}
+        steps:
+          - uses: actions/setup-go@master
+            with:
+              go-version: ${{ matrix.go-version }}
+          - name: Checkout code
+            uses: actions/checkout@master
+          - name: golangci-lint
+            uses: golangci/golangci-lint-action@v3
+            with:
+              version: latest
+              args: --timeout=3m
+    codecov:
+        # golandci-lint need to be run before codecov and succeed
+        needs: golangci-lint
+        name: Code Coverage
+        if: github.repository == 'meshery/meshery-operator'
+        runs-on: ubuntu-22.04
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@master
+          - name: Set up Go
+            uses: actions/setup-go@v2
```

Comment:
I believe `actions/setup-go` is on `v3` going on to `v4`

Reply:
Nice catch, I will replace it to `v4`